data_bounds.py

```diff
@@ -0,0 +1,26 @@
+# List of valid rate selections from high to low in Hz
+VALID_RATES = [120, 10, 1, 0]
```

Comment:
```suggestion
VALID_RATES = (120, 10, 1, 0)
```

Tuple seems more appropriate, as it's closer to immutable

Reply:
I agree, I'll change this